### ISSUE TITLE
learning rate

### ISSUE THREAD

**Issue by usef1371** [YangZeyu95/unofficial-implement-of-openpose#43]
Hi,
I use my own dataset in train.py and step size is 350. in during training , learning rate is decreasing and in middle of steps it's value is 0.0 and loss doesn't change. I try another init value for learning rate also test momentum. but i get an error in line 168,
total_loss, gs_num, summary, lr = sess.run([loss, global_step, merged, learning_rate])
how can i solve this problem?
thanks

**Comment by YangZeyu95** [comment]
you are recommended to use a fix learning rate, 1e-4 will be fine.

**Comment by usef1371** [comment]
@YangZeyu95  thank you. it solves my problem. 
sorry i have another question, i test run.py with a test image from my own dataset that it's single person but i don't get any body, means:             
 bodys = PoseEstimator.estimate_paf(peaks[0], heatmap[0], vectormap[0])  is None.

(line 275 in estimator.py : pafprocess.get_num_humans() = 0 )

what should i do to solve this problem?
final total_loss value with my small dataset is 21.

Thanks

**Comment by YangZeyu95** [comment]
you can check heatmaps and peaks to make sure that the output of the network is correct.

**Comment by usef1371** [comment]
Thank you for your response, i did it and heatmaps and peaks are about correct. but my issue is not solved yet. i don't get any body:
bodys = PoseEstimator.estimate_paf(peaks[0], heatmap[0], vectormap[0]) is None 
i think maybe it is because of multi person . my own dataset is for single person.
Can you guide me?
Thanks

**Comment by YangZeyu95** [comment]
PoseEstimator is used to get multi-person in one image. if you only need one person, just get the index of peaks in peak maps as keypoints positions.

**Comment by usef1371** [comment]
thanks for your reply. 
i am confused beacause in coco dataset there are images with one person. also for evaluation of my results, i need confidence score that it calculated in PoseEstimator. now i dont know how to get scores without running bodys = PoseEstimator.estimate_paf(peaks[0], heatmap[0], vectormap[0]) 
thanks.

**Comment by YangZeyu95** [comment]
It works well in my environment with one person. The problem should be in your training phase. you can use the value in heatmaps as the confidence score.
